bgpd/bgp_clist.c

```diff
@@ -748,6 +748,28 @@ bool community_list_exact_match(struct community *com,
 	return false;
 }
 
+bool community_list_any_match(struct community *com, struct community_list *list)
+{
+	struct community_entry *entry;
+	uint32_t val;
+	int i;
+
+	for (i = 0; i < com->size; i++) {
+		val = community_val_get(com, i);
+
+		for (entry = list->head; entry; entry = entry->next) {
+			if ((entry->style == COMMUNITY_LIST_STANDARD) &&
+			    (community_include(entry->u.com, COMMUNITY_INTERNET) ||
```

Comment:
COMMUNITY_INTERNET is deprecated (in the code), why do we want it here?

Reply:
yes, please remove

Reply:
done